src/Symfony/Component/AssetMapper/Compressor/GzipCompressor.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Compressor;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * Compresses a file using zopfli if possible, or fallback on gzip.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+final class GzipCompressor implements SupportedCompressorInterface
+{
+    use CompressorTrait {
+        compress as baseCompress;
```

Comment:
This should be used as `private baseCompress` instead. We don't want to expose a public method.